src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al

```diff
@@ -0,0 +1,218 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Reflection;
+using System.TestTools.TestRunner;
+
+table 149032 "AIT Test Method Line"
+{
+    DataClassification = SystemMetadata;
```

Comment:
You have marked this as SystemMetaData, but all fields as CustomerData. Would it not make more sense to set the table to CustomerData and the remove the field-level DataClassification?

Reply:
Fixed